algebra/src/bw6_761/curves/g1.rs

```diff
@@ -0,0 +1,113 @@
+use crate::{
+    biginteger::{BigInteger384, BigInteger768},
+    bw6_761::{Fq, Fr},
+    curves::{
+        models::{ModelParameters, SWModelParameters},
+        short_weierstrass_jacobian::{GroupAffine, GroupProjective},
+    },
+    field_new,
+};
+
+pub type G1Affine = GroupAffine<Parameters>;
+pub type G1Projective = GroupProjective<Parameters>;
+
+#[derive(Clone, Default, PartialEq, Eq)]
+pub struct Parameters;
+
+impl ModelParameters for Parameters {
+    type BaseField = Fq;
+    type ScalarField = Fr;
+}
+
+impl SWModelParameters for Parameters {
+    /// COEFF_A = 0
+    #[rustfmt::skip]
+
+    const COEFF_A: Fq = field_new!(Fq, BigInteger768([0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]));
+
+    /// COEFF_B = 6891450384315732539396789682275657542479668912536150109513790160209623422243491736087683183289411687640864567753786613451161759120554247759349511699125301598951605099378508850372543631423596795951899700429969112842764913119068298
```

Comment:
Is `COEFF_B == -1`? If so it may be cleaner to just report it as follows:

```suggestion
    /// COEFF_B = -1
```